tox.ini

```diff
@@ -1,7 +1,7 @@
 [flake8]
 ignore = E265,E501
 max-line-length = 100
-max-complexity = 10
+max-complexity = 11
 exclude = pinax/stripe/migrations/*,docs/*
```

Comment:
Hm....

Reply:
:)

Reply:
I was already getting concerned with the excessive branching.  I wonder if there is a way to simplify rather than increase the complexity.